### PR TITLE
[NCL-1810] Use PNC ide-config style

### PR DESCRIPTION
**Pull request by thescouser89**
Style from: https://raw.githubusercontent.com/project-ncl/ide-config/master/eclipse/jboss-java-formatter.xml

Note: no new code was added in this commit. It's just application of the
codestyle

### CODE REVIEW

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java [hunk 1]
```diff
@@ -47,34 +47,26 @@ public void submit(Runnable task) {
...
-            Consumer<TaskParameters<T>> task,
-            Optional<T> initialLastResult,
-            Consumer<T> onResult,
-            BackOffRunnableConfig backOffRunnableConfig
-            ) {
+    public <T> void subscribe(Subscription subscription,
+                              Consumer<TaskParameters<T>> task,
+                              Optional<T> initialLastResult,
+                              Consumer<T> onResult,
+                              BackOffRunnableConfig backOffRunnableConfig) {
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: Previous alignment is better when long names are used.
Alsthogh the final bracket should be in the same line.

File: src/main/java/org/jboss/pnc/bifrost/Config.java [hunk 2]
```diff
@@ -14,14 +14,16 @@
...
+                    defaultValue = "100")
     int defaultSourceFetchSize;
 
-    @ConfigProperty(name = "bifrost.maxSourceFetchSize", defaultValue = "1000")
+    @ConfigProperty(name = "bifrost.maxSourceFetchSize",
+                    defaultValue = "1000")
     int maxSourceFetchSize;
 
-    @ConfigProperty(name = "bifrost.sourcePollThreads", defaultValue = "4")
+    @ConfigProperty(name = "bifrost.sourcePollThreads",
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/Config.java]: This one is not long enough for wrapping ?

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java [hunk 3]
```diff
@@ -88,7 +80,10 @@ public void submit(Runnable task) {
         backOffRunnable.setRunnable(internalTask);
         backOffRunnable.setCancelHook(() -> unsubscribe(subscription, UnsubscribeReason.NO_DATA_FROM_SOURCE));
 
-        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable, 0, backOffRunnableConfig.getPollIntervalMillis(), TimeUnit.MILLISECONDS);
+        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable,
+                                                                          0,
+                                                                          backOffRunnableConfig.getPollIntervalMillis(),
+                                                                          TimeUnit.MILLISECONDS);
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: That's what I meant in the previous comment.

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java [hunk 4]
```diff
@@ -47,34 +47,26 @@ public void submit(Runnable task) {
         executor.submit(task);
     }
 
-    public <T> void run(
-            Consumer<TaskParameters<T>> task,
-            Optional<T> initialLastResult,
-            Consumer<T> onResult) {
+    public <T> void run(Consumer<TaskParameters<T>> task, Optional<T> initialLastResult, Consumer<T> onResult) {
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: This one is a bit long, should we set the width according to this github view ?

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java (hunk 1 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: hmm what long names are you referring to?

File: src/main/java/org/jboss/pnc/bifrost/Config.java (hunk 2 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/Config.java]: I will look into it again, but I think eclipse doesn't provide such a setting

File: src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java (hunk 1 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java]: This I can do. There's an eclipse setting for that.

File: src/main/java/org/jboss/pnc/bifrost/Config.java (hunk 2 above)
Comment (thescouser89) [reply src/main/java/org/jboss/pnc/bifrost/Config.java]: Given the alignment you want for methods and functions, we can do the same thing for annotations, wrapping if necesasry

**Comment by thescouser89** [comment]
@matejonnet Please review the new formatting. Thanks for all your input! :)

**Comment by thescouser89** [comment]
@matejonnet ping ping ping please review! :)

File: src/main/java/org/jboss/pnc/bifrost/source/ClientFactory.java [hunk 5]
```diff
@@ -41,8 +41,8 @@ public ClientFactory(ElasticSearchConfig config) {
     public RestClient getConnectedClient() throws Exception {
         try {
             List<HttpHost> httpHosts = Arrays.stream(config.getHosts().split(","))
-                    .map(h -> HttpHost.create(h))
-                    .collect(Collectors.toList());
+                                             .map(h -> HttpHost.create(h))
+                                             .collect(Collectors.toList());
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/source/ClientFactory.java]: To many indents.

File: src/main/java/org/jboss/pnc/bifrost/source/ClientFactory.java [hunk 6]
```diff
@@ -55,8 +55,8 @@ public RestClient getConnectedClient() throws Exception {
                     truststore.load(is, config.getKeyStorePass().get().toCharArray());
                 }
                 SSLContextBuilder sslBuilder = SSLContexts.custom()
-                        .loadTrustMaterial(truststore, new TrustSelfSignedStrategy())
-                        .loadKeyMaterial(truststore, config.getKeyPass().get().toCharArray());
+                                                          .loadTrustMaterial(truststore, new TrustSelfSignedStrategy())
+                                                          .loadKeyMaterial(truststore, config.getKeyPass().get().toCharArray());
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/bifrost/source/ClientFactory.java]: To many indents.

File: src/test/java/org/jboss/pnc/bifrost/common/scheduler/SubscriptionsTest.java [hunk 7]
```diff
@@ -36,20 +36,15 @@ public void shouldSubcribeTaskAndRunIt() throws TimeoutException, InterruptedExc
             resultsQueue.add(line);
         };
 
-        Subscription subscription = new Subscription("1", "A", () -> {});
+        Subscription subscription = new Subscription("1", "A", () -> {
+        });
```
Comment (matejonnet) [inline src/test/java/org/jboss/pnc/bifrost/common/scheduler/SubscriptionsTest.java]: No need to  wrap empty body.

File: src/test/java/org/jboss/pnc/bifrost/mock/LineProducer.java [hunk 8]
```diff
@@ -21,13 +21,13 @@ public static Line getLine(Integer lineNumber, boolean last, String ctx, String
...
-                .last(last)
-                .mdc(mdc)
-                .build();
+                   .id(UUID.randomUUID().toString())
+                   .timestamp(Long.toString(System.currentTimeMillis()))
+                   .logger(loggerName)
+                   .message("Message " + lineNumber)
+                   .last(last)
+                   .mdc(mdc)
+                   .build();
```
Comment (matejonnet) [inline src/test/java/org/jboss/pnc/bifrost/mock/LineProducer.java]: To many indents.

File: src/test/java/org/jboss/pnc/bifrost/source/ElasticSearchEmbeddedIT.java [hunk 9]
```diff
@@ -46,13 +46,12 @@
     private String defaultLogger = "org.jboss.pnc._userlog_";
 
     private static ElasticSearchConfig elasticSearchConfig = ElasticSearchConfig.newBuilder()
-        .hosts("http://localhost:9200")
-        .indexes(indexes)
-        .build();
+                                                                                .hosts("http://localhost:9200")
+                                                                                .indexes(indexes)
+                                                                                .build();
```
Comment (matejonnet) [inline src/test/java/org/jboss/pnc/bifrost/source/ElasticSearchEmbeddedIT.java]: To many indents.

**Comment by thescouser89** [comment]
@matejonnet @project-ncl/pnc-team Please review the codestyle changes again and let me know what you think!

**Comment by thescouser89** [comment]
I'll merge it to be able to complete our sprint. However please feel free to provide feedback on the code formatting for further tweaks!

**Pull request merged**